src/Data/Form.purs

```diff
@@ -614,7 +659,8 @@ mvpTargetableInterest =
       { default: Nothing
       , options: val_ (array_ [])
       , resultValue: val_ (array_ [string_ "name"])
-      , results: val_ (array_ [])
+      , results: val_ (array_ [string_ "targetableInterest"])
```

Comment:
So, does using an array here allow us to look for results in multiple places? That's pretty dope, if I'm understanding correctly

Reply:
Ah, no sorry, the array of strings is a path to the results, similar for the value. Can we change something to make that clearer?

Reply:
Ahh okay, that makes sense. This assumes an object path, then? Which I think is a pretty sane assumption, I don't think we need to concern ourselves with supporting a path like `data[0].results`

Reply:
I'm not sure it needs to be clearer...but I suppose we could have a `path_` which is just an alias for `array_` or something 😅 

Or maybe we just call this label `resultsPath` instead of just `results`

Reply:
It doesn't–because we assume all elements are `String`s, but it seems like we ought to be able to handle that case if we assume elements could be `Int`s or String`s:
```PureScript
array_ [string_ "data", int_ 0, string_ "results"]
```

Reply:
Ah true, yeah so at least this let's us add in support for that down the road without really changing syntax ✨